### PR TITLE
fix: full URLs for all product meta open graph images

### PR DESCRIPTION
**Pull request by zchsh**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-zsabsolute-og-images-hashicorp.vercel.app/) 🔎
- [Asana task](https://app.asana.com/0/1100423001970639/1202407276817529/f) 🎟️

## 🗒️ What

Fixes an issue where Open Graph image URLs were not absolute. Using absolute URLs allows services like Twitter to display the Open Graph images in share cards as expected.

### CODE REVIEW

File: src/data/consul.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "image": "/consul/img/og-image.png",
     "icon": [
       {
-        "href": "/consul/_favicon.ico"
+        "href": "https://www.consulproject.io/consul/_favicon.ico"
```
Comment (BRKalow) [inline src/data/consul.json:19]: I think this change should be on the `image` attribute above 👀

File: src/data/sentinel.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
   "metadata": {
     "title": "Sentinel by HashiCorp",
     "description": "Sentinel is a language and framework for policy built to be embedded in existing software to enable fine-grained, logic-based policy decisions.",
-    "image": "/sentinel/img/og-image.png",
+    "image": "https://docs.hashicorp.com/sentinel/img/og-image.png",
```
Comment (zchsh) [inline src/data/sentinel.json:15]: Note: I'm not sure the open graph image for Sentinel will work yet, as I can't seem to access anything in Sentinel's public folder (favicon.svg is the only thing I can test, though) 🤔

Do we need some extra config to expose folders? Eg I can access https://www.consul.io/consul/_favicon.ico, but not https://docs.hashicorp.com/sentinel/favicon.svg . Seems potentially related to `vault-public` / `waypoint-public` naming? cc @BRKalow

Comment (BRKalow) [reply src/data/sentinel.json:15]: @zchsh I think there might be something funky going on with sentinel as we have a redirect in place to send all docs.hashicorp.com traffic to docs.hashicorp.com/sentinel 💭

Comment (zchsh) [reply src/data/sentinel.json:15]: Ah that makes sense... do you think it makes sense to merge this PR as-is, and follow up on Sentinel?

Comment (BRKalow) [reply src/data/sentinel.json:15]: Yeah, that sounds great to me 😃

**Review by BRKalow — APPROVED** [review]
🙇

**Pull request merged**